doc/manual/xcsoar.sty

```diff
@@ -12,7 +12,7 @@
 \makeatletter
 
 % Font selection
-\usepackage{helvet}
+%\usepackage{helvet}
```

Comment:
What's the use of commenting this out instead of just removing it? If we want this back, we can look at the git history.

Reply:
I hesitated between the two possibilities.
Now, commented line is deleted.